infrastructure/src/main/java/org/corfudb/infrastructure/ServerHandshakeHandler.java

```diff
@@ -118,21 +118,21 @@ public void channelRead(ChannelHandlerContext ctx, Object m) throws Exception {
         // Store clientID as a channel attribute.
         ctx.channel().attr(clientIdAttrKey).set(clientId);
         log.info("channelRead: Handshake validated by Server.");
-        log.debug("channelRead: Sending handshake response: Node Id: {} Corfu Version: {}",
+        log.trace("channelRead: Sending handshake response: Node Id: {} Corfu Version: {}",
                 this.nodeId, this.corfuVersion);
 
         CorfuMsg handshakeResponse = CorfuMsgType.HANDSHAKE_RESPONSE
                 .payloadMsg(new HandshakeResponse(this.nodeId, this.corfuVersion));
         ctx.writeAndFlush(handshakeResponse);
 
         // Flush messages in queue
-        log.debug("channelRead: There are [{}] messages in queue to be flushed.", this.messages.size());
+        log.trace("channelRead: There are [{}] messages in queue to be flushed.", this.messages.size());
         while (!messages.isEmpty()) {
             ctx.writeAndFlush(messages.poll());
         }
 
         // Remove this handler from the pipeline; handshake is completed.
-        log.info("channelRead: Removing handshake handler from pipeline.");
+        log.trace("channelRead: Removing handshake handler from pipeline.");
```

Comment:
You can add 1 info/debug log saying handshake succeeded. This is anyways a rare event - when a node/client reconnects.